JavaScript/Citation_Text_Converter/README.md

```diff
@@ -0,0 +1,69 @@
+# Citation Text Converter
+
+## BACKGROUND:
+
+Probably the most boring yet important task for any researcher, reviewer, manuscript writer is to provide references/citations to the prior art or the referenced work of other research papers, articles or bibliography which is often referred as "Citation Text". 
+
+---
+
+## WHAT IF:
+
+* By just the DOI (unique Id) of a Research Paper, one can get the desired Citation Text?
+* By just the BibTeX format (usually available easily) one can generate the Citation Text?
+* Customized format templates are also supported?
+* Comes with well-known inbuilt language options, format styles etc.?
+* With just few lines of easily integrable code?
+
+---
+
+## INTRODUCTION:
+
+Citation Text converter can easily convert your DOI, BibTeX and other well known formats to the Citation Text, just by integrating this script. And not just only Citation Text, but this also supports other conversions such as DOI to BibTeX, BibTeX to DOI etc.
+
+---
+
+## GET STARTED:
```

Comment:
You can remove these parts and just add the commands of running the script via NPM

Reply:
Ohkay!